### ISSUE TITLE
Surpressing Warnings and Errors

### ISSUE THREAD

**Issue by bugybunny** [KevM/tikaondotnet#126]
I‘m sorry, this is a question but I didn‘t know where to post otherwise.

I get a lot of 
`ERROR Can't read the embedded Type1C font EX_CFF_TradeGothic_CondEighteenBold
WARN  Using fallback font TimesNewRomanPS-BoldMT for EX_CFF_TradeGothic_CondEighteenBold`
when reading a PDF. I would like to surpress those warnings/exceptions as they slow down and I don‘t need to see them. They are not critical for me (at least it seems so, the extracted text is still correct). 
I googled and there are possibilities to surpress those thrown PDFBox exceptions by changing the loglevel (like this for example https://stackoverflow.com/a/11466785/4040068) Well, now my question is how I can do this in TikaOnDotNet as I have no config files for log4j. Any idea?

**Comment by Andrei19612015** [comment]
Still have that problem in ver. 1.17.1. Help anybody!
To bugybunny: Are You solved it?

**Comment by bugybunny** [comment]
> To bugybunny: Are You solved it?

Unfortunately not. I did not look further. I was just annoyed by the log output but in the end it did not affect the text extraction.

**Comment by Andrei19612015** [comment]
Thanks for the quick reply, Marco!KevM didn't replied so long to You so it is main reason I decided go to other soft.My customer need in best support. Sincerely, Andrei. 29.03.2021, 22:33, "Marco Syfrig" ***@***.***>: To bugybunny: Are You solved it?Unfortunately not. I did not look further. I was just annoyed by the log output but in the end it did not affect the text extraction.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by bugybunny** [comment]
> KevM didn't replied so long to You so it is main reason I decided go to other soft

Yeah, looking at the commits and other issues this project seems really dead.

**Comment by Andrei19612015** [comment]
I like iText7, but it has very bad doc.30.03.2021, 14:08, "Marco Syfrig" ***@***.***>:

KevM didn't replied so long to You so it is main reason I decided go to other soft

Yeah, looking at the commits and other issues this project seems really dead.

**Comment by KevM** [comment]
This project is not so much dead as limited by the unsupported IKVM.

**Comment by Andrei19612015** [comment]
Thanks. Kevin!  19.05.2021, 17:31, "Kevin Miller" ***@***.***>: This project is not so much dead as limited by the unsupported IKVM.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
